src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Controllers/AdminController.cs

```diff
@@ -1,51 +1,54 @@
-﻿using DevAdventCalendarCompetition.Models;
+﻿using System;
+using System.Globalization;
+using DevAdventCalendarCompetition.Models;
 using DevAdventCalendarCompetition.Services.Interfaces;
 using DevAdventCalendarCompetition.Services.Models;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
-using System;
 
 namespace DevAdventCalendarCompetition.Controllers
 {
     [Authorize(Roles = "Admin")]
     public class AdminController : Controller
     {
-        private readonly IAdminService _adminService;
-        private readonly IBaseTestService _baseTestService;
+        private readonly IAdminService adminService;
+        private readonly IBaseTestService baseTestService;
 
         public AdminController(IAdminService adminService, IBaseTestService baseTestService)
         {
-            _adminService = adminService;
-            _baseTestService = baseTestService;
+            this.adminService = adminService;
+            this.baseTestService = baseTestService;
         }
 
         public ActionResult Index()
         {
-            var tests = _adminService.GetAllTests();
-            ViewBag.DateTime = DateTime.Now;
-            ViewBag.DateTimeUtc = DateTime.UtcNow;
+            var tests = this.adminService.GetAllTests();
+            this.ViewBag.DateTime = DateTime.Now;
+            this.ViewBag.DateTimeUtc = DateTime.UtcNow;
 
-            return View(tests);
+            return this.View(tests);
         }
 
         public ActionResult AddTest()
         {
-            return View();
+            return this.View();
         }
 
         [HttpPost]
         public ActionResult AddTest(TestVm model)
         {
-            if (ModelState.IsValid)
+            if (this.ModelState.IsValid)
             {
-                var dbTest = _baseTestService.GetTestByNumber(model.Number);
+#pragma warning disable CA1062 // Validate arguments of public methods
+                var dbTest = this.baseTestService.GetTestByNumber(model.Number);
```

Comment:
please add checking TestVm model in the first line of this method (if null, throw ArgumentNullException) and remove pragma warning